Docs/user-guides/reference/02-working-with-openl-tables/03-table-types/03-spreadsheet-table/06-spreadsheet-result-output-customization.md

```
@@ -0,0 +1,217 @@
+  "Value_BankQualitativeIndexCalculation": 0.9,
+  "Description_IsAdequateNormativeIndexCalculation": "Calculate Indices B, B1, B2, B3 accoding to Financial Data and Quality Indicators",
+  "Value_IsAdequateNormativeIndexCalculation": 1,
+  "Description_BankRating": "Bank Rating R = B x B1 x B2 x B3",
+  "Value_BankRating": 0.85,
+  "Description_BankRatingGroup": "Calculate Bank Rating Group",
+  "Value_BankRatingGroup": "R2",
+  "Description_LimitIndex": "Calculate Limit Index Kl",
+  "Value_LimitIndex": 1,
+  "Description_Limit": "Max Limit which Bank is Allowed\nL = Kl x Lmax",
+  "Value_Limit": 5000
+}	
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 2

(MD010, no-hard-tabs)